PRINCIPLES.md

```diff
@@ -134,15 +145,15 @@ OCI is not in the CNCF. The OCI project operates according to the norms of inter
 
 ## We Want a Comprehensive Toolchain
 
-_Principle: users don’t need to look beyond the CNCF for cloud native app tooling_
+_Principle: CNCF is the home for all cloud native app tooling_
```

Comment:
I think i like the previous phrasing better. It means the CNCF should provide all required tooling, but not to the point of claiming to host all the cloud native tooling. We had cases of projects applying for sandbox where it was recommended they apply to other foundations - CI, Fintech, ... - and there are cloud native projects living outside the CNCF (Kubeflow, Yunikorn, ...).